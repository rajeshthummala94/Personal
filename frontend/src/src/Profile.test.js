
import React from 'react';
import { shallow } from 'enzyme';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
import MyComponent from '../src/components/Profile/profile';
describe('MyComponent', () => {
    it('Verify the class Name', () => {
        const component = shallow(<MyComponent debug />);
        expect(component).toMatchSnapshot();
    });
});