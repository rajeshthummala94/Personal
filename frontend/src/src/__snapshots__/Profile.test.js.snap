// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent Verify the class Name 1`] = `
<div
  className="container-fluid"
>
  <Redirect
    to="/login"
  />
  <div
    className="card-fluid shadow-sm p-3 mb-5 rounded text-left"
  />
</div>
`;
